### DISCUSSION TITLE
Arm linux support

### DISCUSSION THREAD

**Discussion by nfivez2** [Tyrrrz/DiscordChatExporter#588] [General]
Will there be any support for arm based machine like the raspberry pi in the future?

**Comment by Tyrrrz** [comment]
.NET already supports ARM as far as I know, so if you install the appropriate runtime it should work. The CLI at least.

**Reply by nfivez2** [reply]
Hmm I'll try again in CLI as I have only tried docker, so that's my bad. I'll update as soon as possible.

**Reply by nfivez2** [reply]
It works! Although I might have to set path for regular use, it was pretty easy to use. I hope it gets a arm docker version one day. Thank you for the nice work!
